plugins/osdn/api/types.go

```diff
@@ -1,13 +1,10 @@
 package api
 
 import (
-	knetwork "k8s.io/kubernetes/pkg/kubelet/network"
 	pconfig "k8s.io/kubernetes/pkg/proxy/config"
 )
 
 type OsdnNodePlugin interface {
-	knetwork.NetworkPlugin
-
 	Start() error
 }
```

Comment:
can probably just get rid of OsdnNodePlugin in this case like I did with OsdnMasterPlugin?